SparkPost/2web2ui
# SD-2118 - Removes conditional check for auto warmup enabled when rendering the "Shared Pool" UI option

[SD-2118](https://sparkpost.atlassian.net/browse/SD-2118)

### What Changed
- Removed conditional logic to check for `auto_warmup_enabled` before rendering the "Shared Pool" `<option>` when editing an IP pool

### How To Test
- The code doesn't work locally because of checks for `inSPC()` and `inSPCEU()` - this logic can be temporarily removed and verify that the `auto_warmup_enabled` flag no longer impacts the IP pools "Overflow Pool" select options
- This can be verified at the following route: `account/ip-pools/edit/<ip-pool>`

### To Do
